pkg/hintrunner/zero/zerohint_ec_test.go

```diff
@@ -846,6 +846,47 @@ func TestZeroHintEc(t *testing.T) {
 				check: apValueEquals(&utils.FeltOne),
 			},
 		},
+		"IsZeroNondet": {
+			{
+				operanders: []*hintOperander{},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					err := ctx.ScopeManager.AssignVariable("x", bigIntString("0", 10))
+					if err != nil {
+						t.Fatal(err)
+					}
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newIsZeroNondetHint()
+				},
+				check: apValueEquals(feltUint64(1)),
+			},
+			{
+				operanders: []*hintOperander{},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					err := ctx.ScopeManager.AssignVariable("x", bigIntString("42", 10))
+					if err != nil {
+						t.Fatal(err)
+					}
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newIsZeroNondetHint()
+				},
+				check: apValueEquals(feltUint64(0)),
+			},
+			{
+				operanders: []*hintOperander{},
+				ctxInit: func(ctx *hinter.HintRunnerContext) {
+					err := ctx.ScopeManager.AssignVariable("x", bigIntString("101", 10))
+					if err != nil {
+						t.Fatal(err)
+					}
+				},
+				makeHinter: func(ctx *hintTestContext) hinter.Hinter {
+					return newIsZeroNondetHint()
+				},
+				check: apValueEquals(feltUint64(0)),
+			},
+		},
```

Comment:
isn't this test redundant?

Reply:
agree just one test is enough

the function takes as input a field element so it doesnt make sense to test it with negative values. currently the 3 unit tests test the same thing so just one should be enough

Reply:
Okay. Removing the test. But 2 tests are required, why do you say one @TAdev0  ?